src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -106,7 +76,42 @@ public function refreshCatalogAction(Request $request)
         return new JsonResponse($responseArray, 200);
     }
 
-    public function manageAction($category = null, $keyword = null)
+    private function constructJsonCatalogBodyResponse($modulesProvider, $products)
+    {
+        $formattedContent = [];
+        $formattedContent['selector'] = '.module-catalog-page';
+        $formattedContent['content'] = $this->render(
+            'PrestaShopBundle:Admin/Module/Includes:sorting.html.twig',
+            [
+                'totalModules' => count($products)
+            ]
+        )->getContent();
+        $formattedContent['content'] .=$this->render(
```

Comment:
space missing (I'm always surprised how PHP can handle this kind of stuff ahaha =)